native/src/main/java/io/ballerina/stdlib/sql/nativeimpl/CallProcessor.java

```diff
@@ -176,6 +176,8 @@ private static Object nativeCallExecutable(BObject client, Object paramSQLString
                 return ErrorGenerator.getSQLDatabaseError(e, "Error while executing SQL query: " + sqlQuery + ". ");
             } catch (ApplicationError e) {
                 return ErrorGenerator.getSQLApplicationError(e);
+            } catch (Throwable th) {
+                return ErrorGenerator.getSQLError(th, "Error while executing SQL query: " + sqlQuery + ". ");
```

Comment:
Shall we use `String.format` instead of string concatenation ? Check for other places as well.

```suggestion
                return ErrorGenerator.getSQLError(th, String.format("Error while executing SQL query: %s. ", sqlQuery));
```

Reply:
Fixed in 59a95eb